hack/make.sh

```diff
@@ -96,6 +96,7 @@ fi
 if [ "$DOCKER_EXPERIMENTAL" ]; then
 	echo >&2 '# WARNING! DOCKER_EXPERIMENTAL is set: building experimental features'
 	echo >&2
+	VERSION+="-experimental"
```

Comment:
Ah, here's where it snuck in, @jfrazelle.  I think we should probably just revert this one line, IMO.